src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -354,6 +355,11 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
         binding.ancestorStrict<RsValueParameterList>()?.let { checkDuplicates(holder, binding, it, recursively = true) }
     }
 
+    private fun checkPatIdent(holder: RsAnnotationHolder, patIdent: RsPatIdent) {
+        if (patIdent.pat !is RsPatRest) return
+        SLICE_PATTERNS.check(holder, patIdent, "slice patterns")
```

Comment:
Let's highlight only `..` because:
* only `..` pattern was experimental
* you have to do it in case of rest pattern without binding
* in combination with type hints we get quite weird highlighting
![image](https://user-images.githubusercontent.com/2539310/77147480-6039d000-6a9e-11ea-8b1e-6a69a71dee9d.png)